Documentation/core-repos.md

```diff
@@ -0,0 +1,30 @@
+# .NET Core Repos
+
+There are many repos that make up .NET Core. To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows.
```

Comment:
That feels like passive construction in the sense that the second sentence contains the more important verb or the first.